### PR TITLE
OH-1035 - patient consensus screenshots

### PR DESCRIPTION
**Pull request by goto-eof**

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -1584,6 +1584,13 @@ To modify the anamnesis data of a patient, select the patient in the *_Patient B
 
 image:image247.png[Anamnesis]
 
+====== 8.4.7 Patient consensus
+Consent from a patient is needed regardless of the procedure, whether it's a physical examination or something else. The principle of consent is an important part of medical ethics and international human rights law.
+ In Open Hospital there are three types of Patient consent:
+- Consent to treatment: consent to treatment means a person must give permission before they receive any type of medical treatment, test or examination. This field is mandatory.
+- Administrative consent: administrative consent order means a legal agreement signed by the Director and the owner or operator of a Poultry Feeding Operation.
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]: How did this reappear?   Poultry Feeding Operation

Comment (goto-eof) [reply doc_user/UserManual.adoc]: Sorry :). Done.

File: doc_user/media/image89.png
Comment (mwithi) [inline doc_user/media/image89.png:1]: Please remove the actor's image

File: doc_user/media/image90.png
Comment (mwithi) [inline doc_user/media/image90.png:1]: Please remove the actor's image

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**